harp/_logging.py

```diff
@@ -87,6 +87,7 @@ def _get_logging_level(name: Optional[str], *, default="warning"):
         "harp_apps": {"level": _get_logging_level("harp")},
         "harp_apps.http_client": {"level": _get_logging_level("http_client")},
         "harp_apps.proxy": {"level": _get_logging_level("proxy")},
+        "harp_apps.storage.worker": {"level": _get_logging_level("proxy")},
```

Comment:
Maybe to match how the other logging levels we can set it for the storage app ?

```suggestion
        "harp_apps.storage": {"level": _get_logging_level("storage")},
```

Did not test it yet but it should work as the logging namespaces are built like a tree (harp_apps.storage should apply to harp_apps.storage.worker unless overriden, only question is is there log pollution from root app namespace, I'll have a look later but to be coherent, it seems more logical to define it at app root level and fix the pollution if there are any).

Should also be added to https://docs.harp-proxy.net/en/latest/operate/logging.html